{
  "uid" : "87611e4423eb499",
  "name" : "test_create_trade[BUY_NOW-WAP]",
  "fullName" : "testcase.buyer.test_002_create_trade.TestCreateTrade#test_create_trade",
  "historyId" : "31d841af8fb6478bcfbd5779965b16d4",
  "time" : {
    "start" : 1683884666997,
    "stop" : 1683884666997,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "pymysql.err.OperationalError: (1045, \"Access denied for user 'root'@'1.85.49.19' (using password: YES)\")",
  "statusTrace" : "@pytest.fixture(scope='session', autouse=False)\n    def db_connect():\n>       db_util = DBUtil(host=\"82.156.74.26\", user='root', password='Testfan#123')\n\nconftest.py:47: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\db_study\\db_util.py:16: in __init__\n    self.connect = pymysql.connect(host=host,\n..\\venv\\lib\\site-packages\\pymysql\\connections.py:353: in __init__\n    self.connect()\n..\\venv\\lib\\site-packages\\pymysql\\connections.py:633: in connect\n    self._request_authentication()\n..\\venv\\lib\\site-packages\\pymysql\\connections.py:907: in _request_authentication\n    auth_packet = self._read_packet()\n..\\venv\\lib\\site-packages\\pymysql\\connections.py:725: in _read_packet\n    packet.raise_for_error()\n..\\venv\\lib\\site-packages\\pymysql\\protocol.py:221: in raise_for_error\n    err.raise_mysql_exception(self._data)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\ndata = b\"\\xff\\x15\\x04#28000Access denied for user 'root'@'1.85.49.19' (using password: YES)\"\n\n    def raise_mysql_exception(data):\n        errno = struct.unpack(\"<h\", data[1:3])[0]\n        errval = data[9:].decode(\"utf-8\", \"replace\")\n        errorclass = error_map.get(errno)\n        if errorclass is None:\n            errorclass = InternalError if errno < 1000 else OperationalError\n>       raise errorclass(errno, errval)\nE       pymysql.err.OperationalError: (1045, \"Access denied for user 'root'@'1.85.49.19' (using password: YES)\")\n\n..\\venv\\lib\\site-packages\\pymysql\\err.py:143: OperationalError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "aaget_logger",
    "time" : {
      "start" : 1683884664918,
      "stop" : 1683884664980,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "hasContent" : false,
    "stepsCount" : 0
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1683884664411,
      "stop" : 1683884664917,
      "duration" : 506
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "hasContent" : false,
    "stepsCount" : 0
  }, {
    "name" : "get_buyer_token",
    "time" : {
      "start" : 1683884664980,
      "stop" : 1683884665290,
      "duration" : 310
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "hasContent" : false,
    "stepsCount" : 0
  }, {
    "name" : "worker_id",
    "time" : {
      "start" : 1683884664918,
      "stop" : 1683884664918,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "hasContent" : false,
    "stepsCount" : 0
  }, {
    "name" : "db_connect",
    "time" : {
      "start" : 1683884665291,
      "stop" : 1683884665504,
      "duration" : 213
    },
    "status" : "broken",
    "statusMessage" : "pymysql.err.OperationalError: (1045, \"Access denied for user 'root'@'1.85.49.19' (using password: YES)\")\n",
    "statusTrace" : "  File \"D:\\Program\\Github\\Python\\apistudy\\venv\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\Program\\Github\\Python\\apistudy\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1129, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\Program\\Github\\Python\\apistudy\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 901, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\Program\\Github\\Python\\apistudy\\apiframework\\conftest.py\", line 47, in db_connect\n    db_util = DBUtil(host=\"82.156.74.26\", user='root', password='Testfan#123')\n  File \"D:\\Program\\Github\\Python\\apistudy\\db_study\\db_util.py\", line 16, in __init__\n    self.connect = pymysql.connect(host=host,\n  File \"D:\\Program\\Github\\Python\\apistudy\\venv\\lib\\site-packages\\pymysql\\connections.py\", line 353, in __init__\n    self.connect()\n  File \"D:\\Program\\Github\\Python\\apistudy\\venv\\lib\\site-packages\\pymysql\\connections.py\", line 633, in connect\n    self._request_authentication()\n  File \"D:\\Program\\Github\\Python\\apistudy\\venv\\lib\\site-packages\\pymysql\\connections.py\", line 907, in _request_authentication\n    auth_packet = self._read_packet()\n  File \"D:\\Program\\Github\\Python\\apistudy\\venv\\lib\\site-packages\\pymysql\\connections.py\", line 725, in _read_packet\n    packet.raise_for_error()\n  File \"D:\\Program\\Github\\Python\\apistudy\\venv\\lib\\site-packages\\pymysql\\protocol.py\", line 221, in raise_for_error\n    err.raise_mysql_exception(self._data)\n  File \"D:\\Program\\Github\\Python\\apistudy\\venv\\lib\\site-packages\\pymysql\\err.py\", line 143, in raise_mysql_exception\n    raise errorclass(errno, errval)\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : true,
    "attachmentsCount" : 0,
    "hasContent" : true,
    "stepsCount" : 0
  } ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "testcase.buyer"
  }, {
    "name" : "suite",
    "value" : "test_002_create_trade"
  }, {
    "name" : "subSuite",
    "value" : "TestCreateTrade"
  }, {
    "name" : "host",
    "value" : "DESKTOP-OU7UF4S"
  }, {
    "name" : "thread",
    "value" : "13652-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "testcase.buyer.test_002_create_trade"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "client",
    "value" : "'WAP'"
  }, {
    "name" : "way",
    "value" : "'BUY_NOW'"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "87611e4423eb499.json",
  "parameterValues" : [ "'WAP'", "'BUY_NOW'" ]
}